packages/@sanity/desk-tool/src/panes/document/timeline/timelineItem.styled.tsx

```diff
@@ -12,104 +12,99 @@ export interface TimelineItemProps {
   isHovered: boolean
 }
 
-export const IconWrapper = styled(Flex)(({theme}: IconWrapperProps) => {
-  const borderColor = theme.sanity.color.base.skeleton?.from
-
-  return css`
-    --timeline-hairline-width: 1px;
-    position: relative;
-    z-index: 2;
-    margin: 0;
-    padding: 0;
-
-    &::before {
-      position: absolute;
-      content: '';
-      height: 100%;
-      width: var(--timeline-hairline-width);
-      background: ${borderColor};
-      top: 0;
-      left: calc((100% - var(--timeline-hairline-width)) / 2);
-      z-index: 1;
-    }
-  `
-})
-
-export const Root = styled(MenuItem)(({state = 'enabled', isHovered, theme}: TimelineItemProps) => {
-  const {color} = theme.sanity
-
-  const selectedState = color.button.default.primary.enabled
+export const StyledMenuItem = styled(MenuItem)(({theme}: {theme: Theme}) => {
   return css`
-    position: relative;
-    min-width: 244px;
-
-    ${state === 'selected' &&
-    css`
-      --card-bg-color: ${selectedState.bg};
-      --card-fg-color: ${selectedState.fg};
-      --card-muted-fg-color: ${selectedState.muted};
-      --card-border-color: ${selectedState.bg};
-      &:not([data-selection-bottom='true']) {
+    &:not([data-selection-bottom='true'][data-selection-top='true']) {
+      &[data-selection-top='true'] {
+        border-bottom-left-radius: 0;
+        border-bottom-right-radius: 0;
+      }
+      &[data-selection-bottom='true'] {
         border-top-left-radius: 0;
         border-top-right-radius: 0;
       }
-    `}
+    }
 
-    ${state === 'withinSelection' &&
-    css`
-      border-bottom-left-radius: 0;
-      border-bottom-right-radius: 0;
-      box-shadow: 0px 3px 0px 0px var(--card-bg-color);
-      &:not([data-selection-top='true']) {
+    &:not([data-selection-top='true']) {
+      &[data-selection-within='true'] {
         border-radius: 0;
       }
+    }
 
-      ${IconWrapper} {
-        &::before {
-          background: var(--card-hairline-soft-color);
-        }
+    &:not([disabled], [data-selected]) {
```

Comment:
I know that the names tend to be pretty direct and straightforward but I'd suggest adding comments to the top of these sections explaining what they are